function/package.json

```diff
@@ -16,6 +16,7 @@
     "test-report": "mkdir -p test/reports/unit && nyc ava --tap --timeout=30s --verbose test/*test.js > test/reports/unit/ava.tap"
   },
   "devDependencies": {
+    "acorn": "^6.1.1",
```

Comment:
acorn was introduced as a peer dependency (which are not automatically installed), so I've added it to the dependency list.